fineract-provider/src/main/java/org/apache/fineract/portfolio/tax/service/TaxReadPlatformServiceImpl.java

```diff
@@ -49,7 +48,7 @@ public class TaxReadPlatformServiceImpl implements TaxReadPlatformService {
     private final JdbcTemplate jdbcTemplate;
     private final AccountingDropdownReadPlatformService accountingDropdownReadPlatformService;
 
-    @Autowired
+
     public TaxReadPlatformServiceImpl(final JdbcTemplate jdbcTemplate,
```

Comment:
Please use Lombok annotations to get rid of the constructor. Let's clean up when we have the opportunity.

Reply:
done, updated.